tgui/packages/tgui/interfaces/CrewCredits/type.ts

```diff
@@ -82,8 +82,45 @@ export interface ScoreItemProps {
   value: string;
 }
 
+export interface CitationTabData {
+  tickets: CitationTargetData[];
+  fines: CitationTargetData[];
+}
+
+export interface ReportTabData {
+  reports: ReportData[];
+}
+
+export interface CitationTargetListProps {
+  title: string;
+  citation_targets: CitationTargetData[];
+}
+
+export interface CitationTargetData {
+  name: string;
+  citations: string[];
+}
+
+export interface ReportData {
+  issuer: string;
+  pages: PaperData[];
+  multi: number;
+}
+
+interface PaperData {
+  name?: string;
+  sizeX: number;
+  sizeY: number;
+  text: string;
+  paperColor: string;
+  stamps: [string, number, number, number]
+}
```

Comment:
Refactoring PaperSheet to a .ts file is outside of the scope of your PR, but this type should definitely live over with it rather than here.